src/topic_parameters/topic_parameters.yaml

```diff
@@ -0,0 +1,14 @@
+PLAY_FACE_TOPIC: cordial/face/play
+PLAY_GESTURE_TOPIC: cordial/gesture/play
+_MOUSE_EVENT_TOPIC: cordial/gui/event/mouse
+_KEYPRESS_EVENT_TOPIC: cordial/gui/event/keypress
+_NEW_SERVER_EVENT_TOPIC: cordial/gui/event/new_server
+_USER_RESPONSE_TOPIC: cordial/gui/user_response
+_DISPLAY_TOPIC: cordial/gui/display
+_USER_PROMPTED_TOPIC: cordial/gui/prompt
+_SAY_TOPIC: cordial/say
+_PLAY_WAV_FILE_TOPIC: cordial/sound/play/file_path/wav
+_PLAY_FACE_TOPIC: cordial/face/play
+_PLAY_GESTURE_TOPIC: cordial/gesture/play
+_IS_FACE_IDLE_TOPIC: cordial/face/is_idle
+_IS_GO_TO_SLEEP_TOPIC: cordial/sleep
```

Comment:
It would be better to make this nested. [Here](https://github.com/robotpt/cordial/blob/master/src/cordial_manager/config/config.yaml) is an example.

Reply:
That was my initial thought, however, I realized that we want to use the variable naming convention to find topic names. As I understand it, the nesting can unravel the topics, but wouldn't that lookup the opposite (use topic name to lookup a value)?

Reply:
```
cordial_face:
  play_topic: "cordial/face/play"

cordial_gui:
  play_topic: "cordial/gui/play"

...

```

The script that publishes the topic is the one that is used to lookup the topic name.